### PR TITLE
Group5

### PR DESCRIPTION
**Pull request by wnoland90**
Roger Gordon
Will Noland

### CODE REVIEW

File: language_detector/main.py [hunk 1]
```diff
@@ -2,8 +2,50 @@
...
 
-
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
+    '''makes the paragraph break into a list'''
+    word_list = text.split()
+
+    # How many languages do we have?
```
Comment (corginomics) [inline language_detector/main.py:11]: I don't think this comment is necessary. It's intuitive enough to figure out

File: language_detector/main.py [hunk 2]
```diff
@@ -2,8 +2,50 @@
...
+    
+        '''this will take the find out how many languages we have and put them 
+        in a range'''
+        for y in range(lang_count):
+            
+            '''Stands for current language:''' 
+            current_lang = LANGUAGES[y]
+            
+            '''calls in the external list lang_list'''
+            #global lang_list
```
Comment (corginomics) [inline language_detector/main.py:35]: You could delete line 34 and 35 for cleanliness

File: language_detector/main.py [hunk 3]
```diff
@@ -2,8 +2,50 @@
...
+            if x in current_lang['common_words']:
+                
+                '''once it matches they add the count of the of the language'''
+                lang_list[current_lang['name']] += 1 # Eg: lang_list["Spanish"] += 1
+                
+    # Now we have a list of languages and a count of the number of times their
+    # common word were found in our lang_list dictionary
+    
+    # Time to figure out which language has the highest value
+    ranking = sorted(lang_list, key=lang_list.get, reverse=True)
```
Comment (Metallidog) [inline language_detector/main.py:48]: max() also uses the key argument. 

```
ranking = max(lang_list, key=lang_list.get)
```

Also there's no reason to create another variable in the namespace, you can just return the solution from here.

```
return max(lang_list, key=lang_list.get)
```

**Comment by Metallidog** [comment]
I suggest taking a look at [PEP8](https://www.python.org/dev/peps/pep-0008/#comments) and [PEP257](https://www.python.org/dev/peps/pep-0257/)
You're using docstrings as block comments and it makes your code hard to read.  Your nested for loop and if statement on lines 25-42 really should be grouped together instead of broken up with all those comments.

**Comment by gorrog** [comment]
Thanks @Metallidog and @corginomics for the feedback. You are right about the comments littering the code. We put these in to aid each other as me and @Landl3ss are in completely different time zones (9 hours apart) so weren't able to complete the solution together. We should have removed these before making the pull request. Will make sure that is done next time!

**Pull request closed without merging**